Library/Formula/tig.rb

```diff
@@ -43,10 +43,18 @@ class Tig < Formula
   def install
     system "./autogen.sh" if build.head?
     system "./configure", "--prefix=#{prefix}", "--sysconfdir=#{etc}"
-    system "make", "install"
+    system "make", "install", "sysconfdir=#{prefix/"share/tig/examples"}"
```

Comment:
The problem is that `make install` will copy `tigrc` to `sysconfdir`. Tig does not currently provide a way to disable installation of `tigrc`, but to simplify the Homebrew formula it might make sense to add it.